llarp/service/auth.cpp

```diff
@@ -18,6 +18,24 @@ namespace llarp::service
     return itr->second;
   }
 
+  std::string
+  AuthResultDescription(AuthResult auth)
+  {
+    switch (auth)
+    {
+      case AuthResult::eAuthAccepted:
+        return "accepted";
+      case AuthResult::eAuthRejected:
+        return "rejected";
+      case AuthResult::eAuthPaymentRequired:
+        return "payment required";
+      case AuthResult::eAuthRateLimit:
+        return "rate limit hit";
+      default:
+        return "unknown";
+    }
+  }
```

Comment:
I'm thinking that there needs to be an "other" option here which could be returned with a short explanatory message.  I can imagine a bunch of random things a paid exit might want to return, e.g. "daily bandwidth exceeded" or "account suspended" or ...

Reply:
Or perhaps: AuthResult has Accepted/Rejected, with an optional reason string that could carry the reason info.

Reply:
added reason info.